.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
```

Comment:
Yes, I got some pb during the rebase and lost the merge commit during the pre-commit hook. Not sure why github is showing the wrong diff though. It should still merge properly as the conflicts were minimal.